cli/tools/doc.rs

```diff
@@ -159,6 +168,7 @@ pub async fn doc(flags: Flags, doc_flags: DocFlags) -> Result<(), AnyError> {
             kind_with_drilldown:
               deno_doc::html::DocNodeKindWithDrilldown::Other(node.kind),
             inner: std::sync::Arc::new(node),
+            drilldown_parent_kind: None,
```

Comment:
I'm not sure about this one. @crowlKats ?

Reply:
yep, thats correct